src/Features/ERDDAP/Superlatives/index.tsx

```diff
@@ -0,0 +1,170 @@
+/**
+ * Superlatives, or what platforms experiencing the most extreme conditions right now
+ */
+import * as React from "react"
+import { Link } from "react-router-dom"
+import { Card, CardBody, CardHeader, Col, Row } from "reactstrap"
+
+import { useUnitSystem } from "Features/Units"
+import { converter } from "Features/Units/Converter"
+import { UnitSystem } from "Features/Units/types"
+import { paths } from "Shared/constants"
+import { round } from "Shared/math"
+import { anHourAgoRounded } from "Shared/time"
+import { urlPartReplacer } from "Shared/urlParams"
+
+import { UsePlatforms } from "../hooks"
+import { PlatformFeature, PlatformTimeSeries } from "../types"
+import { conditions } from "../utils/conditions"
+
+const waveHeight = new Set(conditions.waveHeight)
+const windSpeed = new Set(conditions.windSpeed)
+
+/**
+ * Superlatives, or what platforms experiencing the most extreme conditions right now
+ */
+export const Superlatives: React.FunctionComponent = () => {
+  const unitSystem = useUnitSystem()
+
+  const lastHour = anHourAgoRounded()
+
+  return (
+    <UsePlatforms>
+      {({ platforms }) => <ShowSuperlatives laterThan={lastHour} {...{ platforms, unitSystem }} />}
+    </UsePlatforms>
+  )
+}
+
+interface ShowSuperlativesProps {
+  platforms: PlatformFeature[]
+  unitSystem: UnitSystem
+  laterThan: Date
+}
+
+/**
+ * Show the highest winds and biggest waves for selected platforms
+ *
+ * Only shows if both wind and wave data is available
+ *
+ * @param platforms A selection of PlatformFeatures
+ * @param unitSystem unit system to display with
+ * @param laterThan a date to make sure all the readings are more recent then
+ */
+export const ShowSuperlatives: React.FunctionComponent<ShowSuperlativesProps> = ({
+  platforms,
+  unitSystem,
+  laterThan,
+}) => {
+  const { platform: windPlatform, timeSeries: windTimeSeries } = findHighestCondition(platforms, laterThan, windSpeed)
+  const { platform: wavePlatform, timeSeries: waveTimeSeries } = findHighestCondition(platforms, laterThan, waveHeight)
+
+  if (windPlatform && windTimeSeries && wavePlatform && waveTimeSeries) {
+    return (
+      <Card style={{ marginTop: "1rem" }}>
+        <CardHeader>
+          <h5>Latest Conditions</h5>
+        </CardHeader>
+
+        <CardBody>
+          <Row>
+            <Col>
+              <HighestConditions
```

Comment:
Codacy found an issue: ['HighestConditions' was used before it was defined.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6227039)